app/assets/stylesheets/application.scss

```diff
@@ -288,11 +89,19 @@
     margin-right: 8px;
     }
 
+  .panelzz {
+    background-color: rgba(0,0,0,0.5);
+  }
+
   .panelz {
     background-color: rgba(0,0,0,0.3);
     padding-top: 20px;
   }
 
+  .paneled {
+    background:rgba(255,255,255,0.30);
```

Comment:
Commas in function arguments should be followed by a single space<br>Colon after property should be followed by one space